lib/destroy.js

```diff
@@ -28,6 +42,6 @@ function destroy (state) {
 
     delete state.account
 
-    return clone(accountProperties)
-  })
+    return Promise.resolve(clone(accountProperties))
+  }
 }
```

Comment:
Yes I think I understand, it's pretty interesting I have not seen this before. It looks like if the user is signed in you are chaining two `.then()`. I'm assuming it's possible to chain them.